src/traversing.js

```diff
@@ -145,7 +145,7 @@ jQuery.each( {
 		return siblings( elem.firstChild );
 	},
 	contents: function( elem ) {
-        if ( nodeName( elem, "iframe" ) ) {
+        if ( nodeName( elem, "iframe" ) || nodeName( elem, "object" ) ) {
```

Comment:
I wonder if we should just be checking `typeof elem.contentDocument !== "undefined"` or something to that effect. The check for a specific element is how the regression occurred. Are there any other elements where this might apply?

Reply:
Looks like it's applet, frame, iframe, and object.

Reply:
Good point, thanks!
I'll use that check and work on tests for those tags too.

Also, I'm seeing that the [test for Node 8](https://travis-ci.org/jquery/jquery/jobs/368238987) is failing because the route for the svg is returning a 404.
Any advice on best practices for avoiding this?

Reply:
Is there a specific reason why you're using `baseURL` in the test? Usually a relative reference like `data/1x1.svg` should work.

Reply:
I followed how it was done in other tests. Actually the problem was that I didn't add the `svg` extension to the karma config 😅.

I just applied the change you mention. However, due to the lack of browser support for `applet` nowadays, I wasn't able to prepare a test for this tag.